layouts/partials/scripts.html

```diff
@@ -1,11 +1,8 @@
-
-<script src="/js/popper-1.14.3.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
-<script src="/js/bootstrap-4.3.1.min.js" integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM" crossorigin="anonymous"></script>
-
-{{ if .Site.Params.mermaid.enable }}
-<script src="/js/mermaid.min.js" crossorigin="anonymous"></script>
-
-{{ end }}
+<script src="/js/jquery-3.6.0.min.js" integrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" crossorigin="anonymous"></script>
+<script src="/js/popper-1.16.1.min.js" intregrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
```

Comment:
What's the source for this file? (I can't get the subresource integrity check to come out right).

Reply:
Are these appropriate steps to doublecheck SRI?

1. Get the `.min.js` file from the CD.
2. For this file, generate SRI using `create-sri` and the `openssl ...` methods.
3. Go to ./static/js folder and repeat steps for local `.min.js` files.
4. Verify generated SRI values match integrity field in the code.

I tried it with jquery and they seem to line up.

---

Checking Jquery SRI with the following steps. Generates SRI.
```
Get the file:
curl -o jquery-3.6.0.min.js https://code.jquery.com/jquery-3.6.0.min.js

Check SRI:
➜  ~ create-sri jquery-3.6.0.min.js
sha384 encryption result:
sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK

Check SRI:
➜  ~ cat jquery-3.6.0.min.js | openssl dgst -sha384 -binary | openssl base64 -A
vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK%
```
---

Change to ./static/js folder to check SRI for jquery-3.6.0.min.js. 
```
➜  js git:(bugMermaid) create-sri jquery-3.6.0.min.js
sha384 encryption result:
sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK

➜  js git:(bugMermaid) cat jquery-3.6.0.min.js | openssl dgst -sha384 -binary | openssl base64 -A
vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK%
```

In ./partials/head.html
```
<script src="/js/jquery-3.6.0.min.js" intregrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" crossorigin="anonymous"></script>
```
In ./partials/scripts.html
```
<script src="/js/jquery-3.6.0.min.js" integrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" crossorigin="anonymous"></script>
```